integration-tests/emulated/tests/assets/asset-hub-kusama/src/tests/swap.rs

```diff
@@ -287,3 +287,133 @@ fn cannot_create_pool_from_pool_assets() {
 		);
 	});
 }
+
+#[test]
+fn pay_xcm_fee_with_some_asset_swapped_for_native() {
+	let asset_native = asset_hub_kusama_runtime::xcm_config::KsmLocationV3::get();
```

Comment:
lets [avoid using suffixed](https://github.com/polkadot-fellows/runtimes/pull/187#discussion_r1507063883) stuff like `KsmLocationV3` as much as possible, you can convert `KsmLocation` to desired XCM version:
```suggestion
	let asset_native = asset_hub_kusama_runtime::xcm_config::KsmLocation::get()
		.try_into()
		.expect("conversion works");
```

if does not compile then:
```
	let asset_native =  v3::Location::try_from(asset_hub_kusama_runtime::xcm_config::KsmLocation::get())
			.expect("conversion works");
```